sdk/servicebus/service-bus/perf-tests.yml

```diff
@@ -31,17 +31,17 @@ PackageVersions:
   '@azure/logger': source
 
 Tests:
-- Test: send-event-batch
+- Test: send-messages-batch
```

Comment:
In Python SB perf tests, we differentiate between topics/subscriptions and queues for sending and receiving. I think we should include in the name what is being used. Would something like `send-queue-messages-batch`, `receive-queue-messages`, and `recieve-queue-messages-batch` be okay (assuming that they are using queues)?

Reply:
Do we want to perf test both queues and topics?
Workflow is exactly the same for both queues and topics in terms of the SDK.

Feels more like an integration test if we are testing both.

Reply:
I prefer to do it from a completeness pov because even though the workflow is same, the numbers might not be. For ex. we see a difference in throughput on the send side for queue & topics, the former being double, receives are roughly the same.